arkouda/logger.py

```diff
@@ -340,3 +337,13 @@ def disableVerbose(logLevel: LogLevel = LogLevel.INFO) -> None:
     """
     for logger in loggers.values():
         logger.disableVerbose(logLevel)
+
+
+@typechecked
+def write_log(log_msg: str, log_lvl: LogLevel = LogLevel.INFO):
+    """
+    Allows the user to write custom logs.
+    """
```

Comment:
could we add some of the usual docstrings stuff (`Parameters`, `Returns`, `Examples`)?